reference/configuration/assetic.rst

```diff
@@ -50,7 +50,7 @@ Full Default Configuration
                 some_filter:                 []
             workers:
                 # see https://github.com/symfony/AsseticBundle/pull/119
-                # Cache can also be busted via the framework.templating.assets_version
+                # Cache can also be busted via the framework.assets.version
```

Comment:
do you think a note should be added here as well? https://github.com/symfony/symfony/blob/2.7/UPGRADE-2.7.md I found a lot of changes are passing unreported.